src/views/forms/TransferForm.tsx

```diff
@@ -355,12 +359,16 @@ export const TransferForm = ({
           type={InputType.Number}
           onChange={({ floatValue }: NumberFormatValues) => onChangeAmount(floatValue)}
           value={amount ?? undefined}
-          slotRight={renderFormInputButton({
-            label: stringGetter({ key: STRING_KEYS.MAX }),
-            isInputEmpty: size?.usdcSize == null,
-            onClear: () => onChangeAmount(undefined),
-            onClick: () => onChangeAmount(balanceBN.toNumber()),
-          })}
+          slotRight={
+            isUSDCSelected &&
+            balanceBN.gt(0) &&
+            renderFormInputButton({
+              label: stringGetter({ key: STRING_KEYS.MAX }),
+              isInputEmpty: size?.usdcSize == null,
+              onClear: () => onChangeAmount(undefined),
+              onClick: () => onChangeAmount(balanceBN.toNumber()),
+            })
```

Comment:
should we just disable the button if balance is 0?

Reply:
umm i think i'd rather not show the button since it's mostly absent anyways (not shown in native tokens)